Pagnet/ocean-ds-web
# Create vanilla uikit

## Description

### Major changes

- Renames package from `@useblu/ocean-components` to `@useblu/ocean-core` to maintain consistency across other ocean packages;
- Exports only CSS (removes React stuff from this pkg)

### Others

- Fixes CSS import on storybook;
- Adds Gulp to build scss files;
- Adds Stylelint to run into scss files;
- Change folder structure;

### Inspirations

- https://sass-guidelin.es/ (structure, patterns)
- https://github.com/audi/audi-ui (gulp)
- https://github.com/foundation/foundation-sites (gulp, structure)
- https://github.com/TryKickoff/kickoff (structure)
- https://github.com/twbs/bootstrap (structure, coding)

## Types of changes




## How Has This Been Tested?

- Setup application (Follow [this](https://github.com/Pagnet/ocean-ds-web/blob/create-vanilla-uikit/.github/CONTRIBUTING.md#setup) to learn how)
- Run `yarn storybook` and see the stories.

## Screenshots (if appropriate):

![image](https://user-images.githubusercontent.com/3240432/113707860-e11e3680-96b6-11eb-85b5-3942dfd1bdaa.png)

## Checklist:

- [x] My changes work in Chrome, Edge, and Firefox
